src/styles/index.scss

```diff
@@ -1,3 +1,63 @@
+$days-of-week: (
+  mon: 0,
+  tue: 1,
+  wed: 2,
+  thu: 3,
+  fri: 4,
+  sat: 5,
+  sun: 6,
+);
+
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.calendar {
+  width: 707px;
+  display: flex;
+  flex-wrap: wrap;
+
+
+  &__day {
+    padding: 10px;
+    display: flex;
+    box-sizing: border-box;
+    height: 100px;
+    width: 100px;
+    background-color: #eee;
+    border: 1px solid black;
+    gap: 1px;
```

Comment:
The `gap` property is not needed here since `.calendar__day` elements are not direct children of a flex or grid container that would require a gap between them.